src/test/java/uk/nhs/digital/nhsconnect/nhais/service/SequenceServiceTest.java

```diff
@@ -56,4 +58,10 @@ public void When_GenerateIdsForInvalidRecipient_Expect_Exception() {
         assertThrows(SenderAndRecipientValidationException.class, () ->
                 sequenceService.generateInterchangeId("sender", ""));
     }
+
+    @Test
+    public void When_GenerateTransactionIdsForInvalidSender_Expect_Exception() {
+        assertThrows(SenderValidationException.class, () ->
```

Comment:
(Optional) We agreed to use AssertJ - see `assertThatThrownBy()`

Reply:
OK, I will change it :)